src/engine/duel/animations/screen_effects.asm

```diff
@@ -277,11 +277,11 @@ Func_1ce03:
 	jp Func_3bb5
 
 .pointer_table
-	dw Func_190f4         ; DUEL_ANIM_150
-	dw PrintDamageText    ; DUEL_ANIM_PRINT_DAMAGE
+	dw SetScreenForDuelAnimation ; DUEL_ANIM_SET_SCREEN
+	dw PrintDamageText    ; DUEL_ANIM_SHOW_DAMAGE
```

Comment:
I think `DUEL_ANIM_SHOW_DAMAGE` and `DUEL_ANIM_PRINT_DAMAGE` are being used interchangeably.

Reply:
My bad, this is indeed DUEL_ANIM_PRINT_DAMAGE, we should probably have better names to distinguish them